Parse/src/main/java/com/parse/ParseObject.java

```diff
@@ -94,6 +98,14 @@ private static ParseObjectSubclassingController getSubclassingController() {
       return new Builder(className);
     }
 
+    /* package */ static State createFromParcel(Parcel source) {
+      String className = source.readString();
+      if ("_User".equals(className)) {
```

Comment:
I think it’s left as _User everywhere, e.g. [a few lines above](https://github.com/ParsePlatform/Parse-SDK-Android/pull/625/files#diff-8e759e84d8f2c4f7fa6bc715e3786808R95).